templates/konvoy-ui.yaml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1alpha1
+kind: Addon
+metadata:
+  name: konvoy-ui
+  namespace: kubeaddons
+  labels:
+    kubeaddons.mesosphere.io/name: konvoy-ui
+  annotations:
+    appversion.kubeaddons.mesosphere.io/konvoy-ui: "1.47.2"
+    endpoint.kubeaddons.mesosphere.io/konvoy-ui: /ops/portal/
+spec:
```

Comment:
Just a reminder that if no `namespace` is provided in the spec, it will default to `kubeaddons. I believe that's what was intended here however.

Reply:
was intended :D